test/ModuleInterface/actor_availability.swift

```diff
@@ -40,36 +38,34 @@ public actor UnavailableActor {
   // CHECK-NEXT:   get
   // CHECK-NEXT: }
 }
-// CHECK: #endif
 
 // CHECK: @available(macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4, *)
 // CHECK-NEXT: public enum Enum {
 @available(SwiftStdlib 5.2, *)
 public enum Enum {
-  // CHECK:   #if compiler(>=5.3) && $Actors
-  // CHECK-NEXT: @_hasMissingDesignatedInitializers public actor NestedActor {
+  // CHECK-NOT:   #if compiler(>=5.3) && $Actors
+  // CHECK: @_hasMissingDesignatedInitializers public actor NestedActor {
   public actor NestedActor {
     // CHECK: @available(iOS 13.4, tvOS 13.4, watchOS 6.2, macOS 10.15.4, *)
     // CHECK-NEXT: @_semantics("defaultActor") nonisolated final public var unownedExecutor: _Concurrency.UnownedSerialExecutor {
     // CHECK-NEXT:   get
     // CHECK-NEXT: }
   }
-  // CHECK: #endif
 }
 
 // CHECK: extension Library.Enum {
 extension Enum {
-  // CHECK: #if compiler(>=5.3) && $Actors
-  // CHECK-NEXT: @_hasMissingDesignatedInitializers public actor ExtensionNestedActor {
+  // CHECK-NOT: #if compiler(>=5.3) && $Actors
```

Comment:
Nit: In all of these tests there was some effort made to align what the `CHECK` lines were matching for visual clarity. It would be nice if that were preserved.

Reply:
Alright, I've cleaned these up a bit